components/tool/ranking/component.js

```diff
@@ -12,21 +12,64 @@ const Ranking = ({ unit, unitOptions, rankingData }) => {
     return unitOptions.find(option => option.value === unit)?.label;
   }, [unit, unitOptions]);
 
+  const MAX_VOLUME = useMemo(
+    () =>
+      rankingData.reduce(
+        (acc, flow) => {
+          return acc.value < flow.value
+            ? { value: flow.value, formattedValue: flow.formattedValue }
+            : acc;
+        },
+        {
+          value: 0,
+          formattedValue: '',
+        }
+      ),
+    [rankingData]
+  );
+
+  const unitSymbol = useMemo(() => {
+    const value = rankingData?.[0]?.unit;
+    return value;
+  }, [rankingData]);
+
+  const getVolume = value => {
+    const volume = value;
+    console.log((volume * 100) / MAX_VOLUME.value);
+
+    return (volume * 100) / MAX_VOLUME.value;
```

Comment:
Be careful here: `MAX_VOLUME.value` could be 0!

```suggestion
    return MAX_VOLUME.value === 0 ? 0 : (volume * 100) / MAX_VOLUME.value;
```